Admin/AbstractAdmin.php

```diff
@@ -2838,22 +2838,12 @@ public function getDashboardActions()
     }
 
     /**
-     * Setting to true will enable mosaic button for
-     * the admin screen.
-     * Setting to false will hide mosaic button for
+     * Setting to true will enable mosaic button for the admin screen. Setting to false will hide mosaic button for
      * the admin screen.
      */
-    public function setIsShowMosaicButton($isShow)
+    public function setIsShowMosaicButton($isShown)
```

Comment:
This method should also be renamed (to `setShowMosaicButton` ?). In fact, I think 3 methods are not necessary. The following method would be enough :

```
final public function showMosaicButton($showButton)
{
    if ($showButton) {
        $this->listModes['mosaic'] = array('class' => 'fa fa-th-large fa-fw');
    } else {
        unset($this->listModes['mosaic']);
    }
}
```

I removed the cast to bool, because that is exactly what an if statement does,
isn't it?